src/firemon/netstats.c

```diff
@@ -154,6 +154,7 @@ static void print_proc(int index, int itv, int col) {
 		// the sandbox doesn't have a --net= option, don't print
 		if (cmd)
 			free(cmd);
+		free(name);
 		return;
 	}
```

Comment:
```suggestion
		// the sandbox doesn't have a --net= option, don't print
		free(name);
		if (cmd)
			free(cmd);
		return;
	}
    free(name);
```

Free in reverse order and do `free(name)` early.

Reply:
Done